diff --git a/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle b/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
index 4bb66ea..5ea3d7d 100644
--- a/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
+++ b/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
@@ -7,8 +7,8 @@ ext {
 
 buildscript {
     repositories {
-        jcenter()
         google()
+        mavenCentral()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:8.2.1'
@@ -45,7 +45,6 @@ android {
 
 repositories {
     google()
-    jcenter()
     mavenCentral()
 }
 
